src/hooks/screen/useProfileScreenHook.js

```diff
@@ -1,9 +1,10 @@
 import * as React from 'react';
 import netInfo from '@react-native-community/netinfo';
+
 import ProfileRepo from '../../service/repo/profileRepo';
-import useMyProfileFeedContextHook from '../context/useMyProfileFeedContext';
 import StorageUtils from '../../utils/storage';
 import useCoreFeed from '../../screens/FeedScreen/hooks/useCoreFeed';
+import useMyProfileFeedContextHook from '../context/useMyProfileFeedContext';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed. While this doesn't affect the functionality of the code, it's generally a good practice to maintain a consistent order of imports for better readability and maintainability. However, since there is no specific rule about import order in JavaScript, this change is acceptable.
```diff
- import ProfileRepo from '../../service/repo/profileRepo';
- import useMyProfileFeedContextHook from '../context/useMyProfileFeedContext';
- import StorageUtils from '../../utils/storage';
- import useCoreFeed from '../../screens/FeedScreen/hooks/useCoreFeed';
+ import ProfileRepo from '../../service/repo/profileRepo';
+ import StorageUtils from '../../utils/storage';
+ import useCoreFeed from '../../screens/FeedScreen/hooks/useCoreFeed';
+ import useMyProfileFeedContextHook from '../context/useMyProfileFeedContext';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed